web/nginx.conf

```diff
@@ -0,0 +1,22 @@
+server {
+  listen 80;
+
+  sendfile on;
```

Comment:
I left some of the old docker code  around since it might be used in the future.  I just didn't want to delete code that was written already incase we'll use it in the future. We still haven't really discussed how all this will work exactly so I didn't want to make that decision until later.

Reply:
we should **strictly** not be checking in code into the mater branch that **may** be used in the future.

the code will always be around in your own branch so you can always retrieve it when needed! :)